packages/api-bindings/src/graphql/client.graphql

```diff
@@ -16,7 +16,7 @@ extend type Profile {
 
 extend type Post {
   """
-  @deprecated this will be replaced by collectState
+  @deprecated use hasCollectedByMe instead
```

Comment:
this is not added to generated .ts files, so end-user won't see them in their IDE.